src/components/TabulatorComponent.vue

```diff
@@ -44,8 +97,15 @@ export default class TabulatorComponent extends Vue {
 
   @Watch('options', { deep: true })
   private updateOptions() {
+    const vm = this
+    function customMergeFn(objValue: any, srcValue: any, key: string): any | undefined {
+      if (typeof objValue == "function" && typeof srcValue == "function") {
+        return joint([objValue, srcValue])
+      }
+    }
+
     this.resolvedOptions = {
-      ...this.options,
+      ...mergeWith(this.eventOptions, this.options, customMergeFn),
```

Comment:
We can try something new here, the resolved options will be complex, maybe we can try a [Builder](https://sourcemaking.com/design_patterns/builder) or a [Factory](https://sourcemaking.com/design_patterns/factory_method) to simplify the resolved options construct (and make more easily create the others events)
I will try think in something, let my know if you have some suggestion